SPECS/nss_nis/nss_nis.spec

```diff
@@ -2,7 +2,7 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
```

Comment:
A few things we need to do in spec files when we're adding them to the core `SPECS`, which are not required for `SPECS-EXTENDED`:

- Please make sure to run a linter against the specs.
- Remove any macros specific to only other distros. There might be none for this PR but looks for things like `%if 0%{?fedora}`, `%{?rhel}`, etc.
- Make sure the `%check` sections run their tests correctly and without failures, if present. I think you are already doing that but I'm adding it here as a standard reminder.

Reply:
Done. Thank you so much for pointing out the guidelines!